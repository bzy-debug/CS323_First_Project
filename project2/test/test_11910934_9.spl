int sum(int x, int y){
    return x+y;
}

int test_11910934_9()
{
  int x1 = 5;
  int x4 = sum(x1,x1,x1);
  x1[2] = 4;
  return 0;
}
