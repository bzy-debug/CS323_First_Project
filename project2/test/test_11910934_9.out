Error type 9 at Line 8: invalid argument number for sum, expect 2, got 3
Error type 10 at Line 9: indexing on non-array variable
Error type 5 at Line 9: unmatching type on both sides of assignment
